static/src/stylesheets/module/content/_content.scss

```diff
@@ -724,6 +724,15 @@
     .from-content-api {
         @include f-textSans;
     }
+    .element.element-pullquote{
+      .pullquote-paragraph {
+        font-family: $f-sans-serif-headline;
+        color: $paid-article-brand;
+      }
+    }
+    a.in-body-link--immersive {
```

Comment:
.content--immersive-article .in-body-link--immersive we have this but what i want is to change the colour when its ..content--advertisement-feature .in-body-link--immersive but if I don't use a. then the first one takes priority.